src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.java

```diff
@@ -1032,19 +1019,21 @@ public String getImportFullPath() {
         /**
          * Get import start line number.
          *
-         * @return import start line.
+         * @return import ast start line.
```

Comment:
First sentence of Javadoc has the same sentence. It should be updated too.

Also should this be more like "start line from AST"? We are getting the start of the line so original documentation was not wrong, but we are retrieving it from the AST.

Reply:
Done

Reply:
Done